docs/howto/obsolete-term.md

```diff
@@ -43,11 +43,13 @@ Warning: some ontologies give advance notice on terms that will be obsoleted thr
 
 ![](../images/howtoguides/obsolete/fig6.png)
 
-7. Add any additional axioms needed - this is specific to ontologies and you should consult the conventions of the ontology you are working on. 
+7. Add any additional annotations needed - this is specific to ontologies and you should consult the conventions of the ontology you are working on. 
 
-Examples of additional axioms to add: 
-- rdfs:seeAlso - link to github issue
+Examples of additional annotations to add: 
+- rdfs:seeAlso - link to GitHub issue
 - has_obsolence_reason 
+- add ‘OBSOLETE.’ to the term definition: In the 'Description' window, click on the ```o``` on the right-hand side of the definition entry. In the resulting window, in the Literal tab, at the beginning of the definition, type: ```OBSOLETE.``` 
```

Comment:
I will leave this part to @matentzn - vaguely rmbr leaving this out as it might have been ontology specific, but I also see that you put it under "examples" which is reasonable. Other than that I think it looks good :) thanks @nicolevasilevsky

Reply:
@matentzn I put this on our agenda to talk about tomorrow. No need to review this before then.